tests/ui/simd/not-out-of-bounds.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0511]: invalid monomorphization of `simd_shuffle` intrinsic: SIMD index #0 is out of bounds (limit 4)
+error[E0511]: invalid monomorphization of `simd_shuffle` intrinsic: simd_shuffle index must be a SIMD vector of `u32`, got `[u32; 2]`
```

Comment:
Is this file still testing the right thing now?

Reply:
Ah good catch, it is not!
(We do test the out-of-bounds case in another test, though.)

Reply:
Turns out the `simd_insert` part of this was already not testing the right thing...